### DISCUSSION TITLE
MDN "Product" governance: what about the Product Advisory Board? consulted? Still active? Retirement plan?

### DISCUSSION THREAD

**Discussion by SphinxKnight** [mdn/mdn-community#416] [Announcements]
The MDN PAB (cf. https://github.com/mdn/pab) was supposed to be a body with several browser vendors, organizations.

It seems to me that such a group (esp. given its name *Product* *Advisory* Board) should have been consulted for this (AI addition).

1. Were they informed/consulted?
2. When was the last PAB meeting?
3. When is planned the next PAB meeting as of today (i.e. before this question was asked)?
4. If the PAB is not useful anymore what is the planned course of action to retire it?

**Comment by caugner** [comment]
We have recorded your question as number 29.

**Reply by Ultrabenosaurus** [reply]
@LeoMcA @caugner @Rumyra have MDN abandoned their promise to answer all community questions?

**Comment by SphinxKnight** [comment]
From https://developer.mozilla.org/en-US/blog/ai-explain-postmortem/ "In addition to testing, starting March 2023, we conducted Mozilla-internal demos and communicated that work was in progress on these features to stakeholders and our partners." 
Were the PAB and OWD part of those "partners"? If not, can we know who they were?
